src/asthaSrivastava/arrayBasics/ArrayOperations.java

```diff
@@ -0,0 +1,44 @@
+/*Program 1 : (20 mins)
+Return middle character of max length String from given array.
+input : {"Krishna","Maulik","Aashvi","Technocredits","Harsh"}
+output : c
+
+*/
+package asthaSrivastava.arrayBasics;
+
+public class ArrayOperations {
+
+	String getMaxLengthString(String[] name) { // Program1
+		int maxLength = name[0].length();
+
+		String maxLengthName = name[0];
+		for (int index = 0; index < name.length; index++) {
```

Comment:
as you are storing index 0 as max then you can start for loop from index 1

Reply:
Updated code
![image](https://user-images.githubusercontent.com/43742088/190142430-86d4774a-d585-4a42-890c-05b1e66b015b.png)